domain/src/test/java/org/oppia/android/domain/profile/ProfileManagementControllerTest.kt

```diff
@@ -260,6 +260,84 @@ class ProfileManagementControllerTest {
     assertThat(learnerId).isNull()
   }
 
+  @Test
+  fun testFetchContinueButtonAnimationStatus_noLoggedInProfile_returnsNull() {
```

Comment:
Perhaps this should instead test for a profile ID that's not currently logged in? I'm not sure it makes sense to expose separate APIs for the "not logged in" case (see my comment in ProfileManagementController).

Reply:
done -- test updated for not logged in profile